cetmix_tower_server/models/cx_tower_secret_mixin.py

```diff
@@ -0,0 +1,60 @@
+from odoo import api, fields, models
+
+
+class CxTowerSecretMixin(models.AbstractModel):
+    _name = "cx.tower.secret.mixin"
+    _description = "Mixin for managing secrets"
+
+    secret_ids = fields.Many2many(
+        comodel_name="cx.tower.key",
+        relation="cx_tower_secret_rel",
```

Comment:
Model A inherits this mixin. New table with the name `cx_tower_secret_rel` is created in the database.
Model B inherits this mixin. New table with the name `cx_tower_secret_rel` is created in the database.
Why?
Do not put explicit relation keys in the mixin.

Reply:
ok, understood